rest/route/service.go

```diff
@@ -86,6 +86,7 @@ func AttachHandler(app *gimlet.APIApp, queue amboy.Queue, URL string, superUsers
 	app.AddRoute("/projects/{project_id}/test_stats").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetProjectTestStats(sc))
 	app.AddRoute("/projects/{project_id}/task_stats").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetProjectTaskStats(sc))
 	app.AddRoute("/projects/{project_id}/commit_queue").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetCommitQueueItems(sc))
+	app.AddRoute("/projects/{project_id}/commit_queue/{item}").Version(2).Delete().Wrap(checkUser).RouteHandler(makeDeleteCommitQueueItems(sc))
```

Comment:
`checkUser` validates that the user is logged in. With this PR anyone can delete anyone else's commit queue item. Is user validation under a separate ticket?

Reply:
done.
(made a new middleware)